src/InstallHelperPlugin.php

```diff
@@ -135,17 +135,69 @@ public function onWunderIoDdevDrupalPackageUpdate(PackageEvent $event) {
   }
 
   /**
-   * Copy the config.yaml file and the .ddev/wunderio/core directory to the project.
+   * Copy the config.wunderio.yaml file and the dist/ directory contents to the project.
    */
   private function deployDdevFiles(): void {
-    // Copy over the .lando/core directory.
-    $src_core = "{$this->vendorDir}/" . self::PACKAGE_NAME . '/.ddev/wunderio/core';
-    $dest_core = "{$this->projectDir}/.ddev/wunderio/core";
-    self::rcopy($src_core, $dest_core);
-
-    // Copy over the .lando.base.yml file.
-    $src_base = "{$this->vendorDir}/" . self::PACKAGE_NAME . '/.ddev/config.yaml';
-    self::copy($src_base, $this->projectDir);
+    $dest_dir = "{$this->projectDir}";
+
+    // Clean up old files from project root so we can deploy file removal.
+    // This is not ideal solution as we need to keep track of files to delete -
+    // basically this should cover everything that is in the dist/ directory.
+    $paths_to_delete = [
+      '.ddev/config.wunderio.yaml',
+      '.ddev/commands/',
```

Comment:
We need also project specific commands here but we can't delete these. We should delete .ddev/commands/wunderio-* instead